src/desktop/src/ui/components/input/input.scss

```diff
@@ -540,3 +540,20 @@
         }
     }
 }
+
+.tooltip {
+    background: var(--box-bg) !important;
+    color: var(--box-body) !important;
+    width: auto !important;
+
+    &:after {
+        border-bottom-color: var(--box-bg) !important;
+    }
+}
+
+.addressHidden {
+    visibility: hidden;
+    font-size: 14px;
+    padding-left: 10px;
+    padding-right: 40px;
+}
```

Comment:
I run this by Andrew (Trinity designer) and he suggested using other colors for better contrast, split the address in multiple lines for better readability, use `Source Code Pro` font and some minor layout updates:
```
.tooltip {
    background: var(--bar-bg) !important;
    color: var(--bar) !important;
    width: 324px !important;
    font-family: 'SourceCodePro';
    letter-spacing: 2px;
    word-wrap: break-word;
    line-height: 21px;
    padding: 12px 14px 12px 16px !important;

    &:after {
        border-bottom-color: var(--bar-bg) !important;
    }
}
```

Reply:
updated the tooltip style